alacritty_terminal/src/grid/mod.rs

```diff
@@ -125,7 +123,8 @@ pub struct Grid<T> {
     /// updates this offset accordingly.
     display_offset: usize,
 
-    /// An limit on how far back it's possible to scroll
+    /// Legacy value, however we can't remove it, since it's in all ref-tests
+    #[serde(skip)]
     scroll_limit: usize,
```

Comment:
We shouldn't keep around a field that has no purpose.

Reply:
Agree, so we should update all ref-tests to remove this value.

Reply:
Yeah, we should update all of them just to get rid of it.